### PR TITLE
[GHA] [DOCS] Caches document

### PR DESCRIPTION
**Pull request by akashchi**
### Details:
Created upon #22079, please review only the caches doc in this one.

### Tickets:
 - *121283*

### CODE REVIEW

File: docs/sphinx_setup/_static/images/ci/gha_cache_example.png
Comment (ilya-lavrenov) [inline docs/sphinx_setup/_static/images/ci/gha_cache_example.png:1]: please, put under LFS

Comment (akashchi) [reply docs/sphinx_setup/_static/images/ci/gha_cache_example.png:1]: Why? All the other documentation images seem to be not under LFS. Do you suggest putting those needed for CI docs under LFS?

Comment (ilya-lavrenov) [reply docs/sphinx_setup/_static/images/ci/gha_cache_example.png:1]: image

Comment (akashchi) [reply docs/sphinx_setup/_static/images/ci/gha_cache_example.png:1]: My bad, will use the LFS for new ones.

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**